ludwig/encoders/types.py

```diff
@@ -0,0 +1,9 @@
+from typing import TypedDict
+
+import torch
+
+
+class EncoderOutputDict(TypedDict, total=False):
```

Comment:
It would be good to add a comment about which encoders return which tensors, or that only sequence encoders return `encoder_output_state` and `attentions`.

Reply:
Yeah, your're right! I've added the comments. `attentions` are only used by the ViTLegacy Encoder, which is planned to be removed in 0.8. So this return tensor can then be removed too